packages/core/src/Table/TableRow/ExpandedRowContent.styles.js

```diff
@@ -0,0 +1,22 @@
+import { colors } from '@dhis2/ui-constants'
+import css from 'styled-jsx/css'
+
+export default css`
+    tr > td {
+        border-bottom: 1px solid ${colors.grey300} !important;
+    }
+    tr:hover > td,
+    :global(tr.isExpanded:hover) + tr > td {
+        background-color: ${colors.blue050};
+    }
+    :global(tr.isExpanded:active) + tr > td {
+        background-color: ${colors.blue100};
+    }
+    tr.selected > td {
+        background-color: ${colors.teal050};
+    }
+    :global(tr.selected.isExpanded:hover) + tr > td,
+    tr.selected:hover > td {
+        background-color: ${colors.teal100};
+    }
```

Comment:
Just some styles that make the expanded content row and the previous row look like they are one.